examples/simple/main.tf

```diff
@@ -40,6 +24,25 @@ module "seed_bootstrap" {
   billing_account      = var.billing_account
   group_org_admins     = var.group_org_admins
   group_billing_admins = var.group_billing_admins
-  default_region       = var.default_region
   org_project_creators = var.org_project_creators
+
+  project = {
+    project_id    = var.project_id
+    labels        = {}
+    activate_apis = null # Default activate_apis.
+  }
+
+  state_bucket = {
+    name     = var.state_bucket_name
+    location = var.default_region
+    labels   = {}
+  }
+
+  service_account = {
+    account_id          = var.service_account_id
+    grant_billing_user  = true
+    allow_impersonation = true
+    root_roles          = null # Default root roles.
```

Comment:
I would prefer this was something more explicit, like `org_iam_roles` as root terminology is not used widely in our docs.